### ISSUE TITLE
Move from sourceforge to github

### ISSUE THREAD

**Issue by koppor** [JabRef/jabref#111]
This issue is for tracking the migration to github. In the long run, the complete infrastructure of JabRef should move from sourceforge to github. I would not expect the ticket to be closed before 2017.
- [x] Move homepage 
  - [x] converted to markdown
  - [x] host at github (gh-pages) - Skeletton at https://github.com/JabRef/www.jabref.org
    - respect [using-jekyll-with-pages](https://help.github.com/articles/using-jekyll-with-pages/)
  - [x] add redirects for all `php` files at http://jabref.sf.net
- [x] Move Wiki - Refs https://github.com/JabRef/help.jabref.org/issues/32
  <s>
- [ ] Delete [Discussion forums](https://sourceforge.net/p/jabref/discussion/)
  - Point users to [tex.stackexchange.com](http://tex.stackexchange.com/questions/tagged/jabref)
- [ ] Delete [bug tracker](https://sourceforge.net/p/jabref/bugs/)
  - Handle or create new issue on github
- [ ] Delete [feature tracker](https://sourceforge.net/p/jabref/feature-requests/)
  - Handle or create new issue on github
- [ ] Delete [support requests](https://sourceforge.net/p/jabref/support-requests/)
  - Move content to documentation, FAQ or [tex.stackexchange.com](http://tex.stackexchange.com/questions/tagged/jabref)
    </s>
- [x] Move content from the forums and trackers to the documentation
# <s>Discussion points</s>
1. <s>The discussion forum is quite active. Should we keep the discussion forum, should we get a [disqus](https://disqus.com/) forum or should we just drop it?</s>
2. <s>Deletion seems to be quite hard and destroys links. Should we just turn the source forge things read only</s>
3. <s>What should we do with the [mailing lists](https://sourceforge.net/p/jabref/mailman/)?</s>

**Comment by stefan-kolb** [comment]
I would make both the bug tracker and the feature tracker on SF read-only right now.
We don't want to have any new requests via this channel.

**Comment by ThomasA** [comment]
Re. discussion forum, maybe we should create a Google Group? I know 
other examples of projects using that, e.g. 
https://groups.google.com/forum/#!forum/pywavelets, 
https://groups.google.com/forum/#!forum/pandoc-discuss. You can 
subscribe to them by email as well, so as far as I can see they can also 
serve the purpose of mailing lists.

**Comment by lenhard** [comment]
1. What exactly is the purpose of the forum and how is it different from the mailing lists? Anything that remotely relates to bug reports or feature requests could go into the github issues (at least that gives you the ability to tag, sort, close stuff). Even open discussion can be done there, as we are demonstrating right now. Moreover, we have [gitter](https://gitter.im/JabRef/jabref). I would suggest to turn the forum read-only at some point and make clear what the alternatives are. As suggested by @ThomasA, a google group might be a viable alternative. 

2. Turn as much as possible read-only. It takes some time to replace links. But the earlier legacy things become read-only, the better. The ultimate goal should be deletion.

3. Close them. We have too many communication channels right now.

Apart from that, I fully aggree with @stefan-kolb: close the bug and feature trackers on SF asap. Maybe it is possible to link to github issues from there?

**Comment by stefan-kolb** [comment]
3 . I think the mailing lists are a good channel to reach out to people at the moment, so they should stay open for a while.

**Comment by mlep** [comment]
If we change the services, we have to update the website.

**Comment by koppor** [comment]
Discussion forum such as disqus: This allows voting for good answers. I really like that feature. Users tend to use the forum because it is easer than the mailinglist (?). Regarding gitter, I'm not sure whether we should disable it.

I currently resist to close the trackers on sourceforge as the users get notified by the discussions and I don't want to open 200 issues here on github. Manually opening here if someone works on an issue/feature is OK for me.

**Comment by mlep** [comment]
Just a detail: the page <https://github.com/JabRef/jabref/wiki> ask to report bugs to sourceforge.

**Comment by stefan-kolb** [comment]
@mlep fixed :+1:

**Comment by mlep** [comment]
Radically fixed! I first thought you would have stated on the wiki main page that bugs and feature requests should be submitted as issues in github. Well, deletion is another possibility... ;-)

**Comment by stefan-kolb** [comment]
go fast and break things :bowtie:

**Comment by koppor** [comment]
Regarding the conversion: [Pandoc](http://pandoc.org/) seems to be a helpful tool to convert from html to markdown.

**Comment by stefan-kolb** [comment]
The homepage migration could be done by a small student project. Shouldn't be that hard with jekyll.

**Comment by ThomasA** [comment]
You might not need Pandoc. With jekyll, if no plug-ins are required, you may be able to rely on GitHub's jekyll converting the Markdown on commit.

**Comment by koppor** [comment]
@ThomasA: The current homepage is written using PHP and HTML (see https://github.com/JabRef/htdocs). My wish is to convert that to markdown and then using [GitHub's Jekyll feature](https://help.github.com/articles/using-jekyll-with-pages/) to convert to HTML as you suggested.

**Comment by ThomasA** [comment]
Oh that way. Sounds like Pandoc might actually be helpful for converting the HTML to Markdown.

**Comment by mlep** [comment]
Sourceforge's wiki (http://sourceforge.net/p/jabref/wiki/browse_pages/): what else to move?

**Comment by mlep** [comment]
I think we should also preserve the page http://jabref.sourceforge.net/revisionhistory.php
So as to keep the record of JabRef history.
Note: it is referenced here: http://help.jabref.org/en/RevisionHistory/

**Comment by koppor** [comment]
I fixed the link so that it points to http://help.jabref.org/en/RevisionHistory/.

All old urls will point to the new URLs. This is meant by "add redirects for all php files at http://jabref.sf.net"

**Comment by koppor** [comment]
We now have our discussion forum hosted using [discourse](https://www.discourse.org/) at http://discourse.jabref.org/. All discussion forums on sourceforge should be read-only now. I can't set the mailing list read only. Thus, we have to wait until the users really switch to discourse.

**Comment by lenhard** [comment]
Didn't we make some progress here? The website is completely moved, right?

**Comment by koppor** [comment]
No content has been moved. E.g., integrated in the documentation. This applies for the wiki and the content of the discussion forum at sourceforge. The PHP redirects are also not in place.

Example redirect:
 - https://github.com/JabRef/htdocs/blob/master/contact.php
 - Not in place at various other places: All PHP files in help, eg.., https://github.com/JabRef/htdocs/blob/master/help/BaseFrameHelp.php. Why do I want to have the redirects for each page? Because [Cool URIs don't change](https://www.w3.org/Provider/Style/URI.html). And because there exist deep links on the PHP files. This is blocked by https://github.com/JabRef/help.jabref.org/issues/19.

Help wanted :innocent:

**Comment by lenhard** [comment]
Great work @koppor!
